### ISSUE TITLE
Grabbing results count

### ISSUE THREAD

**Issue by iyerushalmi** [kytrinyx/etsy#106]
Hi, unless I am mistaken, it seems there is no way to grab the number of results in an API query. For example if we run a query on Listings, and call the count method it will only tell us how many listings are returned based on the limit, not the total number of results available. Is there any way to grab the total number of results available (i.e. a number greater than 250)?

**Comment by kytrinyx** [comment]
I don't think this is currently possible with the models like Listing. I've only ever used the DSL described here: https://github.com/kytrinyx/etsy#dsl - I think it should give you back the number of results.
